calibrate_linear_node:
  ros__parameters:
    # X=0 / Y=1 
    direction: 1
    # test distance in meters
    test_distance: 1.0
    # robot speed in meters per second
    speed: 0.3
    # error tolerance to goal distance in meters
    tolerance: 0.03
    # linear correction factor
    odom_linear_scale_correction: 1.0
    # check to start the test
    start_test: False
